### ISSUE TITLE
Content of Subject Name (or what are identity fields)

### ISSUE THREAD

**Issue by dcmgashcisco** [haussli/draft-dahm-tacacs-security#35]
" Peers MUST support checking the certificate identity of their Peer
       for policy purposes, either by matching the certificate fingerprint
       or by matching certificate attributes Subject Alternative Name or
       Subject Name.  By configuration, allowing or denying the connection
       based on the result.."

    I think we need to specify what the identity in these fields is, specifically, I assume we are going to use the DNS hostname?

**Comment by haussli** [comment]
That is lacking, isn't it.

rfc5425 S5.2 seems right on mark for this.  I suggest that we use that, as is.  Improve the introduction of it in the draft, then reference 5425.

Should we do anything about 5425 S5.3 and 5.4 (Unauthenticated server/client)?

**Comment by dcmgashcisco** [comment]
rfc5425 model looks good, I would default to just permit the first option, plus support the internationalized with the ref to rfc5280. 

I would skip (not support) the options with * and wildcard,and local Ip in the subject alt, only because I'm not sure it would be useful, so skip them t keep it to simple minimum, but would be happy to hear other side.

**Comment by haussli** [comment]
Just researching; fwiw rfc1422 S3.3 defines/describes the x.509 certificate fields (OIDs), if that is useful for anyone besides myself.

You are referring to rfc5280 S7, I presume.  Internationalized string handling is ugly.

**Comment by td-tacacs** [comment]
@dcmgashcisco When you say "first option in rfc5425", you refer to section 5.1? 
https://datatracker.ietf.org/doc/html/rfc5425#section-5.1

Agree with John, internationalized string handling is ugly, the less we have to write about it the better :) Simply pointing for it to another RFC sounds right.

**Comment by haussli** [comment]
We discussed today whether we need to specify the contents of *any* certificate fields (Subject, AltSubj, etc) or what the the process should be to check identity beyond verification of the certificate itself and what must an operator have to be able to enforce policy (eg: limit connections by peer name).

We concluded that we do not know these answers.  We need to research this a bit to be sure we understand the process and specify as little as possible.
